---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "EntandoApp"
    EntandoResourceNamespace: "my-namespace"
    EntandoApp: "my-app"
  name: "my-app-73892665"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "5cc45473-7441-4f43-a74d-4b410368f1a9"
action: "FAILED"
count: 1
firstTimestamp: "2021-07-05T11:29:15Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "795754"
  uid: "5cc45473-7441-4f43-a74d-4b410368f1a9"
lastTimestamp: "2021-07-05T11:29:15Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Fake Controller\
  \ Exception. Fix the root cause and then trigger a redeployment by adding the annotation\
  \ 'entando.org/processing-instruction: force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "entando-k8s-entando-k8s-app-con-z9fv4-31-from-build-pack--pcgbx"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-app-controller"
type: "Error"
